data/multiplayer/scenarios/2p_Aethermaw.cfg

```diff
@@ -193,14 +193,14 @@
         {PLACE_IMAGE "units/human-loyalists/pikeman-idle-3.png~FL()~RC(magenta>2)" 6 35}
         {PLACE_IMAGE "units/human-loyalists/shocktrooper-attack-2.png~FL()~RC(magenta>2)" 10 37}
         {PLACE_IMAGE "units/human-loyalists/spearman-death4.png~RC(magenta>2)" 5 37}
-        {PLACE_IMAGE "units/human-loyalists/spearman-attack-se-10.png~RC(magenta>2)" 13 36}
-        {PLACE_IMAGE "units/human-loyalists/spearman-attack-s-6.png~RC(magenta>2)" 14 35}
-        {PLACE_IMAGE "units/human-loyalists/spearman-attack-se-9.png~FL()~RC(magenta>2)" 15 36}
+        {PLACE_IMAGE "units/human-loyalists/spearman-attack-se-2.png~RC(magenta>2)" 13 36}
+        {PLACE_IMAGE "units/human-loyalists/spearman-attack-s-3.png~RC(magenta>2)" 14 35}
+        {PLACE_IMAGE "units/human-loyalists/spearman-attack-se-2.png~FL()~RC(magenta>2)" 15 36}
         {PLACE_IMAGE "units/human-loyalists/pikeman-die-2.png~FL()~RC(magenta>2)" 14 37}
         {PLACE_IMAGE "units/human-loyalists/spearman-idle3.png~RC(magenta>2)" 15 37}
         {PLACE_IMAGE "units/human-loyalists/spearman-death3.png~RC(magenta>2)" 13 37}
         {PLACE_IMAGE "units/human-loyalists/pikeman-die-5.png~RC(magenta>2)" 10 38}
-        {PLACE_IMAGE "units/human-loyalists/general-idle-5.png~RC(magenta>2)" 9 36}
+        {PLACE_IMAGE "units/human-loyalists/marshal-leading.png~RC(magenta>2)" 9 36}
```

Comment:
Maybe rebase and force-push to remove the stuff that's already been merged?

Reply:
Does the image changes already exist on the master branch? (I though you only added them to the 1.18 branch)

Reply:
Oops - I remembered cherry-picking them between branches, and didn't look closely enough at which branches they're already on. I think it makes sense to merge them (have made #8623 for that), so that this PR can become smaller.

Reply:
Rebased PR.